src/rust/engine/fs/store/src/lib.rs

```diff
@@ -229,6 +229,59 @@ impl RemoteStore {
       .await
       .map(|&()| ())
   }
+
+  /// Download the digest to the local byte store from this remote store. The function `f_remote`
+  /// can be used to validate/transform the bytes.
```

Comment:
```suggestion
  /// can be used to validate the bytes.
```
`Bytes` is immutable, so the function can only validate.